### ISSUE TITLE
Launch more than one API request at a time

### ISSUE THREAD

**Issue by tollmanz** [johnbillion/resource-host-monitor#4]
Have waited forever for a batch of 19 domains to be analyzed, I think the number of domains analyzed at a time should be experimented with. I think cranking it up to at least 5 simultaneous analyses would be a good start.

**Comment by johnbillion** [comment]
Yeah you're right. I was being conservative with the rate limiting.

The [SSL Labs API](https://github.com/ssllabs/ssllabs-scan/blob/master/ssllabs-api-docs.md) provides `X-Max-Assessments` and `X-Current-Assessments` headers for determining rate limiting.

**Comment by tollmanz** [comment]
I assumed so. Having that experience of having so many domains queued up wasn't great, hence the suggestion.

Those headers are nice. I suppose if we wanted to, we could get really fancy with ramping up and backing off the number of assessments that are being executed.
